File: jobs/2026-05-23__15-22-33/retry_policy_fixed_py__h7A8YFi/artifacts/user/myproject/activities.py
from temporalio import activity
from temporalio.exceptions import ApplicationError

@activity.defn
async def flaky_task() -> None:
    attempt = activity.info().attempt
    with open("/workspace/attempts.log", "a") as f:
        f.write(f"Attempt {attempt}\n")
    raise ApplicationError("boom")

File: jobs/2026-05-27__01-46-48/retry_policy_fixed_ts__jdTrAPV/artifacts/user/myproject/src/activities.ts
import * as fs from 'fs';
import { Context, ApplicationFailure } from '@temporalio/activity';

export async function flakyTask(): Promise<void> {
  const attempt = Context.current().info.attempt;
  fs.appendFileSync('/workspace/attempts.log', `Attempt ${attempt} failed\n`);
  throw ApplicationFailure.create({
    message: `flakyTask failed on attempt ${attempt}`,
  });
}
